subscriptions.md

```diff
@@ -0,0 +1,297 @@
+          },
+          "ePerson": {
+            "href": "http://localhost:8080/server/api/core/subscriptions/60/ePerson"
+          },
+          "self": {
+            "href": "http://localhost:8080/server/api/core/subscriptions/60"
+          }
+        }
+      }
+    ]
+  },
+  "_links": {
+    "self": {
+      "href": "http://localhost:8080/server/api/core/subscriptions/search/findByEPerson?id=49e5ff01-5467-4acd-be21-9ad7374be929"
+    }
+  },
+  "page": {
+    "size": 20,
+    "totalElements": 1,
+    "totalPages": 1,
+    "number": 0
+  }
+}
+```
+
+Status codes:
+* 200 OK - if the operation succeed
+* 400 Bad Request - if the uuid parameter is missing or invalid
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions. Only system administrators or the user specified in the uuid parameter can use the endpoint
+
+
+#### findByEPersonAndDso
+**GET /api/core/subscriptions/search/findByEPersonAndDso?dspace_object_id={id}&eperson_id={id}>**
+Find all subscription that a person made for a dataSpaceObject
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* eperson_id: mandatory, the uuid of the eperson that benefit of content subscription
+* dspace_object_id: mandatory, the uuid of the DSpaceObject (Community, Collection, Item)
+
+```json
+{
+  "subscriptions": [
+    {
+      "id": 60,
+      "type": "subscription",
+      "subscriptionParameterList" : [ {
+			    "id" : 12,
+			    "name" : "Frequency",
+			    "value" : "Daily"
+		  } ],
+      "subscriptionType": "subscription",
+      "_links": {
+        "dSpaceObject": {
+          "href": "http://localhost:8080/server/api/core/subscriptions/60/dSpaceObject"
+        },
+        "ePerson": {
+          "href": "http://localhost:8080/server/api/core/subscriptions/60/ePerson"
+        },
+        "self": {
+          "href": "http://localhost:8080/server/api/core/subscriptions/60"
+        }
+      }
+    }
+  ],
+  "_links": {
+    "self": {
+      "href": "http://localhost:8080/server/api/core/subscriptions/search/findByEPerson?id=49e5ff01-5467-4acd-be21-9ad7374be929"
+    }
+  },
+  "page": {
+    "size": 20,
+    "totalElements": 1,
+    "totalPages": 1,
+    "number": 0
+  }
+}
+```
+Status codes:
+* 200 OK - if the operation succeed
+* 400 Bad Request - if the dspace_object_id or eperson_id parameters are missing or invalid
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions. Only system administrators or the user specified in the eperson_id parameter can use the endpoint
+
+## Creating a subscription
+**POST /api/core/subscriptions?eperson_id=<:uuid>&dspace_object_id=<:uuid>**
+
+Request body:
+```json
+{
+  "subscriptionType": "Test",
+  "subscriptionParameterList": [
+    {
+      "name": "Frequency",
+      "value": "Daily"
+    },
+    {
+      "name": "Frequency",
+      "value": "WEEKLY"
```

Comment:
What are valid Parameters?  Why can there be two Frequencies of different types?